src/main/scala/com/github/krasserm/ases/serializer/EmittedSerializer.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.krasserm.ases.serializer
+
+import java.io.NotSerializableException
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.Serializer
+import com.github.krasserm.ases.Emitted
+import com.github.krasserm.ases.serializer.EmittedFormats.EmittedFormat
+
+class DelegatingEmittedSerializer(system: ExtendedActorSystem)
+  extends EmittedSerializer(new DelegatingPayloadSerializer(system))
```

Comment:
This would require to change the `EmittedFormat` though as the event must be stored as a byte-array.

```
message EmittedFormat {
  bytes event = 1;
  string emitterId = 2;
  string emissionUuid = 3;
}
```

Correct?